places/prague.md

```diff
@@ -0,0 +1,4 @@
+# Prague, CZ
+
+- Located in the heart of Europe.
+- You can eat anywhere :) It is relatively cheap.
```

Comment:
If it is my first time in Prague, what should I definitely make time to see? @jiricermak